### DISCUSSION TITLE
failed to open camera on windows 11

### DISCUSSION THREAD

**Discussion by liuerji** [ffiirree/Capturer#64] [General]
<img width="500" alt="微信图片_20241224151209" src="https://github.com/user-attachments/assets/96305b79-99ea-4784-a0f5-8bf26ad6646f" />
I install Capturer 1.7.0 and open camera failed  on my laptop,  I dont't know why

**Comment by ffiirree** [comment]
Could you upload the last CAPTURER-E-xxx.log file, it is in the applacation's `logs` folder.

**Reply by liuerji** [reply]
ok, I find the log file , I upload the log  as attachment， please help check it 
[CAPTURER-I-20241224-142609.12660.log](https://github.com/user-attachments/files/18248910/CAPTURER-I-20241224-142609.12660.log)

**Reply by liuerji** [reply]
I saw relatively little information from the log file, especially related to the camera.

**Reply by ffiirree** [reply]
It should be three type logging files, inlcuding `CAPTURE-**I**..xx`, `*-W-*` and `*-E-*`. There should be an `*-E-*` file if something goes wrong. 
You can reproduce the issue to generate the error file.

**Reply by liuerji** [reply]
I reproduced this problem, but I didn’t see the log you mentioned. I upload the video when issue happens

https://github.com/user-attachments/assets/428eb359-e891-4834-948f-6146e0e2da6a

**Reply by ffiirree** [reply]
Please try to restart the APP, I have no idea why the error file wasn't generated.

**Reply by liuerji** [reply]
I opened capturer with super administrator rights, and then the log will be generated. Please see the attachment.
[logs.zip](https://github.com/user-attachments/files/18250060/logs.zip)

**Reply by ffiirree** [reply]
Can the camera be opened by the default camera app? It seems like some properties can not be retrieved

**Reply by liuerji** [reply]
It can be opened using the default camera application.

**Reply by ffiirree** [reply]
I can not reproduce this bug ...
